docs/en/connector/flink-sql/MySQL-CDC.md

```diff
@@ -0,0 +1,78 @@
+# Flink SQL MySQL CDC Connector
+
+## Description
+
+We can use the Flink Mysql CDC Connector allows for reading snapshot data and incremental data from MySQL database. Refer to the [Flink CDC MySQL CDC Connector](https://ververica.github.io/flink-cdc-connectors/release-2.1/content/connectors/mysql-cdc.html#connector-options) for more information.
+
+## Usage
+
+### 1. Flink MySQL CDC Version
+
+flink-sql-connector-mysql-cdc-2.1.1
+
+### 2. prepare data
+
+Start mysql server locally, and create a database named "test" and a table named "test_table" in the database.
+
+The table "test_table" could be created by the following SQL:
+
+```sql
+CREATE TABLE IF NOT EXISTS `test_table`(
+   `id` INT UNSIGNED AUTO_INCREMENT,
+   `class_id` INT,
+   `student_id` INT,
+   `course` VARCHAR(100),
+   PRIMARY KEY ( `id` )
+)ENGINE=InnoDB DEFAULT CHARSET=utf8;
+```
+
+Insert some data into the table "test_table".
+
+### 3. seatunnel config 
+
+Prepare a seatunnel config file with the following content:
+
+```sql
+SET 'table.dml-sync' = 'true';
```

Comment:
I have test that the single quote character "'" would cause the config not work. Would you help double check this?